fw/hpack.c

```diff
@@ -1388,6 +1388,40 @@ tfw_hpack_hdr_set(TfwHPack *__restrict hp, TfwHttpReq *__restrict req,
 	return 0;
 }
 
+static int
+process_h2_trailer_hdr(TfwHttpMsg *hm, TfwStr *hdr, int tag)
+{
+	/*
+	 * RFC 9113 8.1:
+	 *
+	 * Trailers MUST NOT include pseudo-header fields.
+	 */
+	switch (tag) {
+	case TFW_TAG_HDR_H2_STATUS:
+	case TFW_TAG_HDR_H2_METHOD:
+	case TFW_TAG_HDR_H2_SCHEME:
+	case TFW_TAG_HDR_H2_AUTHORITY:
+	case TFW_TAG_HDR_H2_PATH:
+	case TFW_TAG_HDR_ACCEPT:
```

Comment:
I see only ":method", ":scheme", ":authority" , ":path"  pseudo-headers in [8.3.1. ](https://www.rfc-editor.org/rfc/rfc9113#section-8.3.1)[Request Pseudo-Header Fields](https://www.rfc-editor.org/rfc/rfc9113#name-request-pseudo-header-field) and [8.1.2.3.](https://httpwg.org/specs/rfc7540.html#rfc.section.8.1.2.3) [Request Pseudo-Header Fields](https://httpwg.org/specs/rfc7540.html#HttpRequest)

Reply:
I added a comment why this list of headers is used.